packages/backend/src/graphql/schema.gql-to-typescript.ts

```diff
@@ -1,5 +1,19 @@
+import paginate from '@/helpers/pagination'
+import App from '@/models/app'
+import Execution from '@/models/execution'
 import ExecutionStep from '@/models/execution-step'
+import Flow from '@/models/flow'
 import TableMetadata from '@/models/table-metadata'
 
-export type TableMetadataGraphQLType = TableMetadata
+export type AppGraphQLType = App
 export type ExecutionStepGraphQLType = ExecutionStep
+export type TableMetadataGraphQLType = TableMetadata
+
+// TODO: check if there is a better way to express connections
+export type ExecutionStepConnectionGraphQLType = ReturnType<
+  typeof paginate<ExecutionStep>
+>
+export type ExecutionConnectionGraphQLType = ReturnType<
+  typeof paginate<Execution>
+>
+export type FlowConnectionGraphQLType = ReturnType<typeof paginate<Flow>>
```

Comment:
nit: why is the word `Connection` included here ah, can't it be called `ExecutionGraphQLType` for example

Reply:
+ 1